microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -31,14 +31,14 @@ This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s, for example:
 //[id="microshift-4-13-rhel-edge"]
-//=== {op-system-base-full} {op-system-version}
-//this is the plain RHEL attribute and version; do we need to create a RHEL for Edge attribute in the common_attributes or microshift_attributes files?
+//=== {op-system-ostree-first} {op-system-version}
 
-//[id="microshift-4-13-new-feat-based-on-RHEL-Edge"]
+//[id="microshift-4-13-new-feat-based-on-{op-system-ostree}"]
 //==== Placeholder for new feat bases on RHEL Edge
 
-//[id="microshift-4-13-installation"]
-//=== Installation (can also include Upgrade when that is available)
+[id="microshift-4-13-installation"]
+=== Installation
+This release includes integration with the Greenboot health check framework. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble. You can add the optional Greenboot RPM to your installation. For more information, read the xref:../microshift_running_apps/microshift-greenboot.adoc#microshirt-greenboot[GreenBoot documentation]
```

Comment:
```suggestion
This release includes integration with the greenboot health check framework. Greenboot assesses system health and automates a rollback to the last healthy state in the event of software trouble. You can add the optional greenboot RPM to your installation. For more information, read the xref:../microshift_running_apps/microshift-greenboot.adoc#microshirt-greenboot[greenboot documentation].
```

- The documentation that you point to uses "greenboot" in all lowercase except where it starts a sentence, so I followed suit in my edit. 
- Not sure if we should spell out "RPM". I looked it up on the [Red Hat abbreviations page](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki/rh_dictionary_letter_r), and apparently it stands for "RPM Package Manager", which a little odd IMHO.